components/binding-usage-controller/docs/usage-kind.md

```diff
@@ -0,0 +1,33 @@
+# UsageKind
+## Overview
+
+UsageKind allows dynamic, in the runtime, Binding Usage Controller configuration. It allows you to define what resources can be bound with ServiceBinding and how to bind them.
+
+## UsageKind structure
+
+From the technical point of view the binding process modifies target resource by adding labels to the Pod. The UsageKind specifies the field which contains labels added to the Pod and the resource type (group, kind and version). 
+
+## Binding Usage Controller
+
+Binding Usage Controller takes the value of the spec.usedBy.kind field of the binding usage resource, looks for the corresponding usage kind which contains information about the resource to be bound. The controller handles resources even if the specified labelsPath field does not exist.
+
+## Finalizer
+
+UsageKind contains a finalizer which protects UsageKind deletion if they are used. The controller removes the finalizer when the UsageKind is not referenced by any ServiceBindingUsage.
+
+## RBAC settings
+
+The administrator who adds UsageKind must take care of RBAC settings. BUC and ui-api-layer must be allowed to perform needed operations on resources with the kind defined or referenced in the usage kind object.
```

Comment:
``The administrator who adds the UsageKind must take care of the RBAC settings. The BUC and ui-api-layer must be allowed to perform needed operations on the resources, with the type defined in the UsageKind object.``